### ISSUE TITLE
indent spaces  not multiple of 4

### ISSUE THREAD

**Issue by harpomaxx** [harpomaxx/galgoR-package#8]
```
NOTE: Consider 4 spaces instead of tabs; 2 lines (0%) contain
      tabs.
    First 2 lines:
      vignettes/galgoR.rmd:50 - Using the molecular data alone with unsupervi...
      vignettes/galgoR.rmd:51 - Using supervised techniques focusing entirely...
    * NOTE: Consider multiples of 4 spaces for line indents, 355
      lines(10%) are not.
    First 6 lines:
      R/callback-functions.R:17              prefix,
      R/callback-functions.R:18              generation,
      R/callback-functions.R:19              pop_pool,
      R/callback-functions.R:20              pareto,
      R/callback-functions.R:21              prob_matrix,
      R/callback-functions.R:22              current_time) {
    See http://bioconductor.org/developers/how-to/coding-style/
```

**Comment by martinguerrero89** [comment]
tab issue solved in commit 5b567da69d172163757a0a3577b2d1220bf51844

**Comment by martinguerrero89** [comment]
tab issue 2 solved in commit c0c05b9ead7513b27d0a6871b0a5bccd140e0a48
